content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,163 @@ ifdef::backend-html5[]
+
+----
+
+Now you are inside the container, download the 'jenkins.war' file from the official site like.
+
+----
+
+# wget wget http://updates.jenkins-ci.org/download/war/2.176.1/jenkins.war
+
+----
+
+Replace the version with the one that fits to you.
+
+The next step is to move that file and replace the oldest one.
+
+----
+
+# mv ./jenkins.war /usr/share/jenkins/
+
+----
+
+Then change permissions.
+
+----
+
+# chown jenkins:jenkins /usr/share/jenkins/jenkins.war
+
+----
+
+The last step is to logout from the container and restart it.
+
+----
+
+$ docker restart jenkins-docker_1
+
+----
+
+You can verify that update was successful by access to you Jenkins url.
+
+== Kubernetes Concepts:
+
+The Concepts section helps you learn about the parts of the Kubernetes system and the abstractions Kubernetes uses to represent your cluster, and helps you obtain a deeper understanding of how Kubernetes works.
+
+* *link:https://kubernetes.io/docs/concepts/overview/[Overview:]* Kubernetes is a portable, extensible, open source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available.
+
+* *link:https://kubernetes.io/docs/concepts/architecture/[Cluster Architecture:]* The architectural concepts behind Kubernetes.
+
+* *link:https://kubernetes.io/docs/concepts/containers/[Containers:]* Technology for packaging an application along with its runtime dependencies.
+
+* *link:https://kubernetes.io/docs/concepts/windows/[Windows in Kubernetes]*
+
+* *link:https://kubernetes.io/docs/concepts/workloads/[Workloads:]* Understand Pods, the smallest deployable compute object in Kubernetes, and the higher-level abstractions that help you to run them.
+
+* *link:https://kubernetes.io/docs/concepts/services-networking/[Services, Load Balancing, and Networking:]* Concepts and resources behind networking in Kubernetes.
+
+* *link:https://kubernetes.io/docs/concepts/storage/[Storage:]* Ways to provide both long-term and temporary storage to Pods in your cluster.
+
+* *link:https://kubernetes.io/docs/concepts/configuration/[Configuration:]* Resources that Kubernetes provides for configuring Pods.
+
+* *link:https://kubernetes.io/docs/concepts/security/[Security:]* Concepts for keeping your cloud-native workload secure.
+
+* *link:https://kubernetes.io/docs/concepts/policy/[Policies:]* Policies you can configure that apply to groups of resources.
+
+* *link:https://kubernetes.io/docs/concepts/scheduling-eviction/[Scheduling, Preemption and Eviction:]* In Kubernetes, scheduling refers to making sure that Pods are matched to Nodes so that the kubelet can run them. Preemption is the process of terminating Pods with lower Priority so that Pods with higher Priority can schedule on Nodes. Eviction is the process of proactively terminating one or more Pods on resource-starved Nodes.
```

Comment:
```suggestion
* link:https://kubernetes.io/docs/concepts/architecture/[Cluster Architecture]: The architectural concepts behind Kubernetes.

* link:https://kubernetes.io/docs/concepts/containers/[Containers]: Technology for packaging an application along with its runtime dependencies.

* link:https://kubernetes.io/docs/concepts/windows/[Windows in Kubernetes]

* link:https://kubernetes.io/docs/concepts/workloads/[Workloads]: Understand Pods, the smallest deployable compute object in Kubernetes, and the higher-level abstractions that help you to run them.

* link:https://kubernetes.io/docs/concepts/services-networking/[Services, Load Balancing, and Networking]: Concepts and resources behind networking in Kubernetes.

* link:https://kubernetes.io/docs/concepts/storage/[Storage]: Ways to provide both long-term and temporary storage to Pods in your cluster.

* link:https://kubernetes.io/docs/concepts/configuration/[Configuration]: Resources that Kubernetes provides for configuring Pods.

* link:https://kubernetes.io/docs/concepts/security/[Security]: Concepts for keeping your cloud-native workload secure.

* link:https://kubernetes.io/docs/concepts/policy/[Policies]: Policies you can configure that apply to groups of resources.

* link:https://kubernetes.io/docs/concepts/scheduling-eviction/[Scheduling, Preemption and Eviction]: In Kubernetes, scheduling refers to making sure that Pods are matched to Nodes so that the kubelet can run them.
Preemption is the process of terminating Pods with lower priority so that Pods with higher priority can schedule on Nodes.
Eviction is the process of proactively terminating one or more Pods on resource-starved Nodes.
```
same list suggestions as above, if these are steps they can be numbered, otherwise they would not need to be indented so far